src/bcf/report/table_report/create_report_table.rs

```diff
@@ -408,6 +403,41 @@ pub(crate) fn get_ann_description(header_records: Vec<HeaderRecord>) -> Option<V
     None
 }
 
+fn read_bcf_record(
+    info_map: &mut HashMap<String, Vec<tera::Value>>,
```

Comment:
```suggestion
    info_map: &mut HashMap<String, Vec<Value>>,
```

I believe the map contains json values from `serde_json` (e.g. `json!(value)`), not the ones from Tera.

Reply:
Oh, I just took what the editor suggested without double checking :D

Reply:
Thats what what I assumed, sometimes my IDE also tricks me into using the wrong `Value`(s).

Reply:
Turns out that `tera::Value` is just a re-export of `serde_json::Value` so everything would have been right even with `Vec<tera::Value>>`.